notification/app/notification/models/Destination.scala

```diff
@@ -0,0 +1,11 @@
+package notification.models
+
+import models.{UserId, Topic}
+
+object Destination {
+  type Destination = Either[Set[Topic], UserId]
+
+  def apply(userId: UserId): Destination = Right(userId)
+  def apply(topic: Topic): Destination = Left(Set(topic))
+  def apply(topics: Set[Topic]): Destination = Left(topics)
```

Comment:
if at least one topic is required to define Destination, maybe vararg like
```
def apply(topic: Topic, topics: Topic*)
```
will suite better ?